src/components/Button.tsx

```diff
@@ -68,6 +68,40 @@ interface ButtonProps {
   url?: string
 }
 
+const animatedTextWrapper = classnames(
+  display('flex'),
+  overflow('overflow-hidden')
+)
+
+// we get a transition-delay for each character separately so that the letters move one after another
+const getTransitionDelay = (index: number) =>
```

Comment:
Let's name it `getCharacterTransitionDelay` and remove the comment :)

Reply:
Done